codalab_service.py

```diff
@@ -738,6 +782,34 @@ def start_services(self):
                 root=(not self.args.codalab_home),
             )
 
+            if 'CODALAB_ADMIN_EMAIL' in self.compose_env:
```

Comment:
you could write this more compactly as `for property, env_var in [('server/admin_email', 'CODALAB_ADMIN_EMAIL'), ...]:`...